core/tee/tee_rpmb_fs.c

```diff
@@ -1028,7 +1028,7 @@ static TEE_Result tee_rpmb_verify_key_sync_counter(uint16_t dev_id)
 	return res;
 }
 
-static TEE_Result tee_rpmb_write_key(uint16_t dev_id)
+static TEE_Result __maybe_unused tee_rpmb_write_key(uint16_t dev_id)
```

Comment:
This function is only used from `tee_rpmb_write_and_verify_key()` below, can't they be in the same ifdef?